packages/elements/src/components/ino-segment-button/ino-segment-button.scss

```diff
@@ -76,21 +93,22 @@ $left-right-segment-border: 0 24px 24px 24px;
     }
 
     &:hover {
-      @include button.outline-color(var(--segment-button-hover-color));
-      @include button.ink-color(var(--segment-button-hover-color));
-      box-shadow: var(--segment-button-hover-shadow);
+      @include button.container-fill-color(var(--segment-button-hover-color));
+      @include button.ink-color(var(--segment-button-checked-color));
+    }
+    &:focus {
```

Comment:
![elements-ino-segment-button-tabbing](https://user-images.githubusercontent.com/22963121/186677854-7b55f716-5038-45e8-9f67-f36a74bd02f4.gif)
When tabbing through the group, the font stays greyish. According to the design (`focused` state).it should be blue.